web_app.py

```
@@ -3209,7 +3203,7 @@ def on_stage(label: str) -> None:
     def worker() -> None:
         global web_game
         try:
-            # 首条阶段立即入队：生成器可在 WebGame 构造（verify smoke）前就 yield
+            # 首条阶段立即入队：生成器可在 WebGame 构造重活前就 yield（#1195 ≤5s 首见）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)